pyproject.toml

```diff
@@ -0,0 +1,9 @@
+[build-system]
+requires = ["setuptools",
+            "setuptools_scm",
+            "wheel",
+            "cython",
```

Comment:
You should pin Cython to ensure you get consistent source generation.

Reply:
Done